apollo-openapi/pom.xml

```diff
@@ -37,6 +37,10 @@
       <groupId>com.ctrip.framework.apollo</groupId>
       <artifactId>apollo-core</artifactId>
     </dependency>
+    <dependency>
```

Comment:
apollo-common has many dependencies, it's not friendly for those apps that depend on apollo-openapi.
I think it's ok to have some duplications as the logic of UrlUtils is simple and clear.

Reply:
I thought so. but when I extract encode/decode method in UrlUtils, It will use in adminservice/ItemController. but adminservice don't have dependency of openapi,  as the definition of common, maybe is more fit for the position of UrlUtils. so, if I just define UrlUtils in openapi, how should I decode key in adminservice/ItemController?  duplicate the decode method in UrlUtils?  it may not a good maintenance.